src/dev/build/tasks/build_kibana_example_plugins.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+import Fs from 'fs';
+import { REPO_ROOT } from '@kbn/utils';
+import { exec, mkdirp, copyAll, Task } from '../lib';
+
+export const BuildKibanaExamplePlugins: Task = {
+  description: 'Building distributable versions of Kibana example plugins',
+  async run(config, log, build) {
+    const examplesDir = Path.resolve(REPO_ROOT, 'examples');
+    const args = [
```

Comment:
@spalger if you get a moment, do you have any thoughts on uthis?  I could look into migrating the CLI to a function.  This works, logs are a little messy though:

```
   │ info Building /home/jon/dev/kibana/examples/screenshot_mode_example                                 
   │ info $ node ../../scripts/plugin_helpers build --kibana-version=8.0.0-SNAPSHOT                      
   │ info  info deleting the build and target directories                                                
   │ info  info running @kbn/optimizer                                                                   
   │ info  │ info initialized, 0 bundles cached                                                          
   │ info  │ info starting worker [1 bundle]                                                             
   │ info  │ succ 1 bundles compiled successfully after 51.8 sec                                         
   │ info  info copying assets from `public/assets` to build                                             
   │ info  info copying server source into the build and converting with babel                           
   │ info  info compressing plugin into [screenshotModeExample-8.0.0-SNAPSHOT.zip] 
```

Reply:
I don't think we need to worry too much about the logs here, they're accurate and that's enough I think.